## Hi there, welcome to my GitHub profile! ğŸ‘‹

## ğŸ“ Who Am I?

- ğŸ”¬ **Postdoctoral Researcher** at [SnT, Luxembourg](https://www.uni.lu/snt-en/)
- ğŸ“ **Two PhDs** in **Mathematics** & **Computer Science**
- ğŸ§¬ **Bioinformatics Researcher** with expertise in **Metagenomics** and **Microbiome Data Analysis**
- ğŸ“ˆ **Google Certified Data Analyst**
- ğŸ’¡ Specialized in **Algebraic Graph Theory**, **Bioinformatics Pipelines**, **Graph Neural Networks**, & **Network Science**
- ğŸŒ± Currently focusing on **Automation in Bioinformatics**, **Graph Databases**, & **Scalable Systems for Biological Data**

---

## ğŸ› ï¸ Skill Set

My skill set is a fusion of theoretical knowledge and practical expertise, particularly in bioinformatics:

> ğŸ“˜ **Programming Languages:** 
> - **Python**: Advanced usage in bioinformatics, data science, and machine learning with libraries like TensorFlow, Keras, Scikit-learn, Pandas, NumPy, and Biopython.
> - **R**: Utilized for statistical analysis and bioinformatics packages like `ggplot2`, `dplyr`, and `edgeR`.
> - **SQL**: Employed for managing biological databases and querying large datasets.
> - **Bash**: Scripting for automating bioinformatics workflows on Linux platforms.

> ğŸ§¬ **Bioinformatics & Computational Biology:** 
> - Expertise in developing and automating bioinformatics pipelines for processing and analyzing metagenomic and microbiome data.
> - Experience with graph-theoretical approaches for analyzing complex biological networks and applying Graph Neural Networks (GNNs) for predictive and descriptive analytics in omics data.
> - Proficient in tools like Nextflow and Docker for containerizing and scaling bioinformatics workflows.

> â˜ï¸ **Cloud Platforms:** 
> - **AWS & GCP**: Applied cloud services for scalable storage and computation of large biological datasets.

> ğŸ“Š **Data Analysis & Visualization:** 
> - Proficient with Pandas and R for data manipulation and analysis in bioinformatics.
> - Skilled in using visualization tools such as Matplotlib, Seaborn, ggplot2, and Cytoscape to derive insights from complex biological data.

> ğŸ—„ï¸ **Graph Databases:** 
> - **Neo4j**: Applied for storing and querying large-scale biological networks, facilitating advanced graph-based analyses.

> ğŸ”§ **Development Tools:** 
> - **Git**: For version control and collaborative research coding.
> - **Linux**: Comfortable with the Linux environment for development and deployment of bioinformatics applications.

> ğŸ¤– **Natural Language Processing (NLP):** 
> - Experience with NLP techniques for analyzing scientific literature and extracting meaningful information from biological texts.

> ğŸ—„ï¸ **Big Data Technologies:** 
> - Utilized PySpark and Hadoop for handling and analyzing large-scale biological datasets in distributed environments.

---

### ğŸ¤ Connect with Me

- [LinkedIn](https://www.linkedin.com/in/eestaji/)
- [Medium](https://medium.com/@ehsan.estaji)
- **Email**: ehsan.estaji [at] gmail [dot] com

---
